bindings/python/tests/test_construct_named_graphs.py

```
@@ -0,0 +1,451 @@
+    """Iterating the result yields `Quad`s whose `graph_name` is the named graph."""
+    result = _store().query(
+        f"CONSTRUCT {{ GRAPH <{EX}g> {{ ?s ?p ?o }} }} WHERE {{ ?s ?p ?o }}"
+    )
+    quads = list(result)
+    assert len(quads) == 2
+    assert {str(quad.graph_name) for quad in quads} == {f"<{EX}g>"}
+
+
+@pytest.mark.parametrize("format", _QUAD_CAPABLE)
+def test_every_quad_capable_format_round_trips_the_graph(
+    format: purrdf.RdfFormat,
```

Ruff (0.16.2)

[error] Function argument `format` is shadowing a Python builtin

(A002)